src/backend/joanie/core/serializers.py

```diff
@@ -396,24 +396,24 @@ class EnrollmentSerializer(serializers.ModelSerializer):
 
     id = serializers.CharField(read_only=True, required=False)
     course_run = CourseRunSerializer(read_only=True)
+    was_created_by_order = serializers.BooleanField(required=True)
 
     class Meta:
         model = models.Enrollment
-        fields = ["id", "course_run", "is_active", "state"]
+        fields = ["id", "course_run", "is_active", "state", "was_created_by_order"]
         read_only_fields = ["id", "state"]
```

Comment:
I think you should repeat course_run here although it is marked as readonly on the field.
```suggestion
        read_only_fields = ["id", "course_run", "state"]
```